### ISSUE TITLE
Consider offering a way to reset DB at Hibernate ORM level for testing

### ISSUE THREAD

**Issue by emmanuelbernard** [quarkusio/quarkus#583]
reset would rerun the DDL statements and import.sql file.

This is useful if you want a test to be fully isolated from another data wise.

**Comment by cunhazera** [comment]
Will this feature be implemented?

**Comment by emmanuelbernard** [comment]
We would need volunteer to implement it to move forward I think.

**Comment by artkonr** [comment]
Hi. I had a similar issue in my project, so I came up with a simplistic utility, which just picks .sql files and runs them as native queries using EntityManager (see sample repo [https://github.com/artkonr/trivial-reset](trivial example)) I would start them in test classes from BeforeEach/AfterEach methods. I figured it would be convenient to let the user of the utility to decide which .sql is to be run in which test.
It's a simplistic solution and can be made a lot prettier and more user-friendly with .sql detection, customization and other features. 

I reckon there can also be another approach with some kind of user-friendly Flyway wrapper which gets started with the QuarkusTest and imported as test dependency. But then again, it will do pretty much the same as my solution does.
